website/src/views/helpers.ts

```diff
@@ -86,6 +97,23 @@ export const getDateRangeFromSearch = (
     return undefined;
 };
 
+export function getNumberRangeFromSearch(
+    search: URLSearchParams | Map<string, string>,
+    name: string,
+): NumberRange | undefined {
+    const minString = search.get(`${name}From`);
+    const maxString = search.get(`${name}To`);
+
+    const min = minString ? Number(minString) : undefined;
+    const max = maxString ? Number(maxString) : undefined;
+
+    if ((min !== undefined && !Number.isNaN(min)) || (max !== undefined && !Number.isNaN(max))) {
```

Comment:
I just discovered this function. Does it work? According to the docs, it should do the same.
```suggestion
    if (Number.isFinite(min) || Number.isFinite(max)) {
```

Reply:
Good idea. Looks way better now.